feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/section/SearchFilter.kt

```diff
@@ -0,0 +1,79 @@
+package io.github.droidkaigi.confsched2023.sessions.section
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalSoftwareKeyboardController
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import io.github.droidkaigi.confsched2023.model.DroidKaigi2023Day
+import io.github.droidkaigi.confsched2023.model.Lang
+import io.github.droidkaigi.confsched2023.model.TimetableCategory
+import io.github.droidkaigi.confsched2023.model.TimetableSessionType
+import io.github.droidkaigi.confsched2023.sessions.component.FilterCategoryChip
+import io.github.droidkaigi.confsched2023.sessions.component.FilterDayChip
+import io.github.droidkaigi.confsched2023.sessions.component.FilterLanguageChip
+import io.github.droidkaigi.confsched2023.sessions.component.FilterSessionTypeChip
+import kotlinx.collections.immutable.ImmutableList
+
+data class SearchFilterUiState<T>(
```

Comment:
Well, I don't think this is SearchFilterUiState because it is UiState for FilterDayChip. 💭 
But for now I'll merge this since it is better than current implementation 🙏